### DISCUSSION TITLE
default_excluded_apps.txt

### DISCUSSION THREAD

**Discussion by 6wendy9** [Romanitho/Winget-AutoUpdate#742] [Q&A]
Hello, 

What is the best way to update some of the applications in the exluded_apps.txt file, I packaged the app as a win32 app in Intune. I would like for teams and teamviewer to be automatically updated and not skipped.

**Comment by Romanitho** [comment]
Bring your own `excluded_apps.txt` that will override the default list.
However, Teams is not well handled by Winget in the system context and is highly recommended to be excluded.

**Comment by 6wendy9** [comment]
Got it, After pushing out WAU it prompts users for admin credentials to update some applications. These are the parameters I set - STARTMENUSHORTCUT=1 NOTIFICATIONLEVEL=Full UPDATESINTERVAL=Daily USERCONTEXT=1

**Reply by whitephnx1** [reply]
if you look in the log file. if the app that prompted was being ran as user that is why it was prompting. I have found some apps that were installed by user with no admin. the winget tries to install the system version that requires admin using the user side. thus you get the notification.

**Reply by Romanitho** [reply]
by default WAU is configured to install apps with machine scope
